lua/lspconfig/server_configurations/rust_analyzer.lua

```diff
@@ -55,9 +55,7 @@ return {
         or util.root_pattern 'rust-project.json'(fname)
         or util.find_git_ancestor(fname)
     end,
-    settings = {
-      ['rust-analyzer'] = {},
-    },
+    settings = {},
```

Comment:
this table can be removed by default.

Reply:
Thanks for the suggestion, I removed the whole `settings` table and rebased onto the current master.